src/plugins/getPlugins.ts

```diff
@@ -0,0 +1,10 @@
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { createAKSClusterPlugin } from "./createAKS/createAKSClusterPlugin";
+import { GetPluginsCommandResult } from "../types/@azure/AzureAgent";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
💡 Lets add proper type here if we can. and avoid liable linting please. `* eslint-disable @typescript-eslint/no-unused-vars */`